### DISCUSSION TITLE
Replacing window.FPConfig{}; with flyingPages.listen();

### DISCUSSION THREAD

**Discussion by Natizyskunk** [gijo-varghese/flying-pages#7] [Ideas]
I've forked your project and try to make it a little bit more user-friendly.
I've modded it to make the script work without the defer attribute and without having to set the options before the script call. 

Replacing `window.FPConfig{};` before the script call with `flyingPages.listen();` after the script call ([see Usage](https://gitlab.com/Natizyskunk/flying-pages/#usage)).

You can see it on my [Gitlab](https://gitlab.com/Natizyskunk/flying-pages/).

Maybe it could be a nice idea to take my changes and added them to your own github.

**Comment by gijo-varghese** [comment]
@Natizyskunk that's great! Can you open a pull request?

**Reply by Natizyskunk** [reply]
Yes no problem, I just made it :)
